### PR TITLE
demo target databases are not read-only per default

### PR DESCRIPTION
**Pull request by ltclm**
This pull request keeps demo target databases  as read/write databases.

### CODE REVIEW

File: db_master_deploy/deploy.sh [hunk 1]
```diff
@@ -203,8 +203,9 @@ copy_database() {
     # add some metainformation to the copied database as comment
     psql -U pgkogis -d template1 -h localhost -c "COMMENT ON DATABASE ${target_db} IS 'copied from ${source_db} on $(date '+%F %T') with command ${COMMAND} by user ${USER}';" > /dev/null
 
-    # set database to read-only if it is not a _master database
-    if [[ ! ${target} == master ]]; then
+    # set database to read-only if it is not a _master or _demo database
+    REGEX="^master|demo$"
```
Comment (gjn) [inline db_master_deploy/deploy.sh]: Not sure if it should be `^(master|demo)$`, but probably both work.

Comment (ltclm) [inline db_master_deploy/deploy.sh]: both are working. in this case we dont need to capture the matched element.

Comment (gjn) [inline db_master_deploy/deploy.sh]: question is more if the `^` and `$` is part of the pipe selection sets. This, I don't know.

Comment (ltclm) [inline db_master_deploy/deploy.sh]: Not sure if i understand the question `^ $`are indicating the start/end of the string.
The following patterns are working:

``` bash
target=demo && regex="^master|demo$" && if [[ ${target} =~ ${regex} ]]; then echo true; else echo false; fi
true

target=demo && regex="master|demo" && if [[ ${target} =~ ${regex} ]]; then echo true; else echo false; fi
true

target=demo && regex="^(master|demo)$" && if [[ ${target} =~ ${regex} ]]; then echo true; else echo false; fi
true

target=demo && regex="(master|demo)" && if [[ ${target} =~ ${regex} ]]; then echo true; else echo false; fi
true

```

**Comment by AFoletti** [comment]
I would use the third one
`target=demo && regex="^(master|demo)$" && if [[ ${target} =~ ${regex} ]]; then echo true; else echo false; fi
true` 

the actual solution returns true even if the targets are not EXACTLY "master" and "demo". In fact, you can add a suffix to master or a prefix to demo and still got TRUE

**Comment by ltclm** [comment]
ok, i have implemented the third one.

**Pull request closed without merging**